libs/datasets/timeseries.py

```diff
@@ -1031,10 +1031,84 @@ def drop_regions_without_population(
 LOCATION_ID_AGG = "location_id_agg"
 
 
+@dataclass(frozen=True)
+class AverageStaticWeightAggregation:
+    """Information about a timeseries field which is aggregated using a weighted average."""
+
+    # timeseries field that gets aggregated using a weighted average
+    field: FieldName
+    # static field that used to produce the weights
+    scale_factor: FieldName
+
+
+AGGREGATIONS = (
```

Comment:
Maybe this could be something like `WEIGHTED_AGGREGATIONS` or `CUSTOM_AGGREGATIONS`.

Reply:
Changed to DEFAULT_AGGREGATIONS. If you don't find a better way of structuring this we may end up extending the list with other types of aggregations.

Reply:
and then changed again to WEIGHTED_AGGREGATIONS because the default is `sum`.